airflow/Dockerfile

```diff
@@ -95,7 +94,12 @@
     chmod a+x /run-airflow.sh && \
     chmod +x /usr/bin/tini
 
-COPY --from=statsd-exporter --chown=stackable:stackable /bin/statsd_exporter /stackable/statsd_exporter
+# Choosing correct binary with respect to architecture
```

Comment:
Please do this in the builder.
That's also where you included the `tar` dependency.
Please add a comment what it's needed for (even though it seems obvious now!)

Reply:
done